invenio_communities/assets/semantic-ui/js/invenio_communities/settings/profile/index.js

```diff
@@ -302,13 +302,13 @@ class CommunityProfileForm extends Component {
     // create a map with all organizations that are not custom (part of the
     // vocabulary), so that on form submission, newly custom organization input
     // by the user can be identified and correctly sent to the backend.
-    const organizationsNames = [];
-    initialValues.metadata.organizations.map((org) => {
+    let organizationsNames = [];
+    organizationsNames = initialValues.metadata.organizations.map((org) => {
       const isNonCustomOrganization = org.id;
       if (isNonCustomOrganization) {
         this.knownOrganizations[org.name] = org.id;
       }
-      organizationsNames.push(org.name);
+      return org.name;
```

Comment:
The linter was complaining that the result of the map was not being assigned to anything. I can't see what change this is making on the behaviour. In my mind, it is just that instead of pushing a value in each iteration to the organizationNames array, we are creating an array with the values of all iteration and assigning it to the organizationNames.

In testing the community settings viewed worked as I expected.

Reply:
the change should be ok!